<?xml version="1.0"?>
<launch>
    <node name="pylon_camera" pkg="pylon_camera" type="pylon_camera_node" output="screen">
        <param name="camera_frame" value="/pylon_camera" type="string" /> <!-- frame to which the camera is attached -->
        <param name="start_exposure" value="5000" type="int" />
        <param name="use_sequencer" value="false" type="bool" />
        <param name="desired_framerate" value="10.0" type="double" />
        <param name="use_brightness" value="true" type="bool" />
        <param name="mtu_size" value="3000" type="int" /> <!-- only for GigE-Cameras -->
    </node>
</launch>
